src/short-css-class-names/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+* Returns an array with valid CSS class names that are between 1 and 3 chars long.
+*
+* Return value in pseudo code:
+* [
+*    "A",
+*    "B",
+*    ...
+*    "Z__",
+* ]
+**/
+module.exports = function() {
+    "use strict";
+
+    let charRange = require("./char-range"),
```

Comment:
I prefer `let` because `const` is kind of a lie since the object can be mutated. I only use `const` for numbers and strings where it actually works as expected. Showing intent is a good argument for using `const` but when you still have to read the code to be sure that the object really isn't mutated I think it becomes almost dangerous. The road to hell is paved by good intentions, as they say.